examples/css/styles.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 16px;
+}
+
+h1 {
+  font-size: 4rem;
+}
+
+p {
+  text-shadow: 4px 4px 4px blue;
+}
```

Comment:
Why is this file neccessary tho?

Reply:
just for visualization of styling sheets